### PR TITLE
Get Rid of useless IP method

### PR DESCRIPTION
**Pull request by x5engine**
removing the method that saves the IPs of users, for privacy sake!

### CODE REVIEW

File: app/kudos/views.py [hunk 1]
```diff
@@ -128,17 +128,6 @@ def marketplace(request):
     if q:
         title = f'{q.title()} Kudos'
         token_list = token_list.keyword(q)
-        # log this search, it might be useful for matching purposes down the line
-        try:
-            SearchHistory.objects.update_or_create(
-                search_type='kudos',
-                user=request.user,
-                data=request.GET,
-                ip_address=get_ip(request)
```
Comment (danlipert) [inline app/kudos/views.py:137]: @x5engine what do you think of just removing the ip_address portion?

**Comment by thelostone-mc** [comment]
Closing this out ! 
@x5engine if you could get one just removing the IP -> we can get it in ^_^

File: app/kudos/views.py (hunk 1 above)
Comment (x5engine) [reply app/kudos/views.py:137]: yes why not but the whole app has many ip addresses

**Comment by x5engine** [comment]
sure  @thelostone-mc

**Pull request closed without merging**